python/mrchem/validators.py

```diff
@@ -318,13 +319,27 @@ def validate_cavity(self):
 
         # the centers of the spheres are the same as the atoms
         if self.cavity_mode == "atoms":
+
+            def radiusNotFound(x):
+                # This raises an exception in the list comprehension if the radius is not valid.
+                raise ValueError(
+                    "The vdw-radius of element {x} is not defined in the mantina set".format(x)
```

Comment:
f-strings should be preferred:
```suggestion
                    f"The vdw-radius of element {x} is not defined in the mantina set"
```